### PR TITLE
Feat/favorites in profile2

### PR DESCRIPTION
**Pull request by Ayisha-sifna**

### CODE REVIEW

**Comment by andrew-bierman** [comment]
Does this work on your end ?

**Comment by Ayisha-sifna** [comment]
yes

File: client/components/user/ProfileContainer.js
Comment (andrew-bierman) [inline client/components/user/ProfileContainer.js:1]: Please fix error when creating a favorite.

Error: Validation failed
```
    at /Users/andrewbierman/Dropbox/AppAcademy/PackRat/PackRat/server/node_modules/celebrate/lib/celebrate.js:95:19
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  details: Map(1) {
    'body' => [Error [ValidationError]: "packId" is required] {
      _original: [Object],
      details: [Array]
    }
  },
  [Symbol(celebrated)]: true
}
Finished POST /user/favorite 400
```

File: client/components/user/ProfileContainer.js [hunk 1]
```diff
@@ -84,13 +80,18 @@ export default function ProfileContainer() {
...
-        <Box
-          style={{ width: "100%", height: "100%" }}
-          space={4}
-          alignItems="center"
-        >
-          <UserDataContainer data={packsData} type="packs" />
-          <UserDataContainer data={tripsData?.trips} type="trips" />
+        <Box style={styles.mainContentContainer}>
+          <Box style={styles.userDataContainer}>
+            <UserDataContainer data={favoritesData} type="favorites" />
```
Comment (andrew-bierman) [inline client/components/user/ProfileContainer.js:85]: What do we show if user does not have any favorites? Current button does not link to anything.

**Review by andrew-bierman — CHANGES_REQUESTED** [review]

**Comment by andrew-bierman** [comment]
Profile page also allows cards from one section to overflow onto another section, need to fix

**Pull request closed without merging**